addons/sale/models/sale.py

```diff
@@ -1390,10 +1390,9 @@ def _compute_untaxed_amount_to_invoice(self):
                 # amount and not zero. Since we compute untaxed amount, we can use directly the price
                 # reduce (to include discount) without using `compute_all()` method on taxes.
                 price_subtotal = 0.0
-                if line.product_id.invoice_policy == 'delivery':
-                    price_subtotal = line.price_reduce * line.qty_delivered
-                else:
-                    price_subtotal = line.price_reduce * line.product_uom_qty
+                umo_qty_to_consider = line.qty_delivered if line.product_id.invoice_policy == 'delivery' else line.product_uom_qty
```

Comment:
```suggestion
                uom_qty_to_consider = line.qty_delivered if line.product_id.invoice_policy == 'delivery' else line.product_uom_qty
```
At least name the variable correctly (even if SIG gives a bad example in 46c4e6fc56518f00d69892e7e715b8877b26e201 :smile: )

Reply:
Woooops